### ISSUE TITLE
Allow us to use HTML in Invoice Terms

### ISSUE THREAD

**Issue by ririko5834** [InvoicePlane/InvoicePlane#758]
Please allow us to use HTML in Invoice Terms. You can't make text bold or add links now, HTML is not rendered correctly.

**Comment by clockwiseq** [comment]
I support this request.  Let me talk with the team to see if we can get this one added to the list of enhancements as it seems like a fairly simple change.

**Comment by ririko5834** [comment]
Will be the HTML rendered in both online invoices, PDF invoices and in email template placeholders?

**Comment by clockwiseq** [comment]
> Will be the HTML rendered in both online invoices, PDF invoices and in email template placeholders?

Yes, should be simple change to render HTML instead of outputting text in both places.

**Comment by ririko5834** [comment]
Offtopic question: why this repo does not have enabled discussions for support and questions? Also can you check my other issues?

**Comment by clockwiseq** [comment]
The repo was restricted during the transfer of parties from the original author to us.  We simply did not have the development resources available to keep up or track all of the issues that were being posted.  We hope in the very near future to have said resources available to work on these items and can open it back up so that everyone can post them.  We generally handle support/questions/requests in the [community](https://community.invoiceplane.com).

**Comment by anondev-sudo** [comment]
When you put url to the invoice terms, it is not displayed as link, just text and users need to copy it to view it. Also you cant make text bold there. HTML or Markdown support there will be really useful
